feconf.py

```diff
@@ -643,3 +643,17 @@ def get_empty_ratings():
 
 CSRF_PAGE_NAME_CREATE_EXPLORATION = 'create_exploration'
 CSRF_PAGE_NAME_I18N = 'i18n'
+
+# pylint: disable=line-too-long
+# The following are all page descriptions for the meta tag.
+ABOUT_PAGE_DESCRIPTION = 'Oppia is an open source learning platform that connects a community of teachers and learners. You can use this site to create 1-1 learning scenarios for others on thare created through modules or explorations.'
+CREATE_PAGE_DESCRIPTION = 'Let us learn from you. Create lessons through explorations and share your knowledge with the community.'
+DASHBOARD_PAGE_DESCRIPTION = 'Keep track of the lessons you have created, as well as feedback from learners.'
+FORUM_PAGE_DESCRIPTION = 'Engage with the Oppia community and discuss questions, bugs, explorations;, and make requests.'
```

Comment:
punctuation issues after 'explorations'. How about:

Engage with the Oppia community: discuss questions, bugs, and explorations in the forum.